### PR TITLE
ES6 module reassignments

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Reassigns ES6 modules based on what people have the bandwidth to complete.

I have removed entries for modules that have outstanding Pull Requests, so this may cause conflicts for those lucky people (mainly @gustavpursche). Sorry about this.

For those that requested a bunch of modules ( @regiskuckaertz, @sndrs, @gustavpursche), I've assigned 10 modules. If anybody wants any more, you can pull them from the `Unassigned` section at the bottom. There are currently ~18~ 16 unassigned modules.

Thanks for all your help with this epic task!

@desbo @akash1810 @jfsoul @philwills @svillafe @rich-nguyen @NataliaLKB 

## What is the value of this and can you measure success?

ES6 all the things ✨ 

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Tested in CODE?

No

### CODE REVIEW

File: tools/es5to6.json [hunk 1]
```diff
@@ -1,122 +1,113 @@
...
-  "Philip Wills": [],
-  "Santiago Villa Fernandez": [],
-  "Jamie Byers": []
-}
\ No newline at end of file
+    "Regis Kuckaertz": [
+        "bootstraps/enhanced/accessibility.js",
+        "bootstraps/enhanced/common.js",
+        "bootstraps/enhanced/crosswords.js",
+        "lib/atob.js",
```
Comment (gustavpursche) [inline tools/es5to6.json]: This doesn't exist anymore (now part of pf.io).

Comment (SiAdcock) [inline tools/es5to6.json]: Lucky old @regiskuckaertz! I'll give him something from the vaults instead 😱

File: tools/es5to6.json [hunk 2]
```diff
@@ -1,121 +1,112 @@
...
-    "projects/common/modules/experiments/affix.js"
+  "Sam Desborough": [
+    "projects/common/modules/experiments/affix.js",
+    "projects/common/modules/experiments/tests/paid-content-vs-outbrain.js"
   ],
-  "davidfurey": [],
-  "jranks123": [
+  "Akash A": [
+    "projects/common/modules/video/metadata.js",
+    "projects/common/modules/video/onward-container.js"
```
Comment (gustavpursche) [inline tools/es5to6.json]: I'm working on this now. Can be removed.

File: tools/es5to6.json [hunk 3]
```diff
@@ -1,121 +1,112 @@
...
+    "bootstraps/enhanced/facia.js",
+    "bootstraps/enhanced/football.js",
+    "bootstraps/enhanced/gallery.js",
     "projects/common/modules/crosswords/cell.js",
     "projects/common/modules/crosswords/classNames.js",
     "projects/common/modules/crosswords/clues.js",
     "projects/common/modules/crosswords/comments.js",
     "projects/common/modules/crosswords/confirm-button.js",
+    "projects/common/modules/discussion/activity-stream.js",
+    "projects/common/modules/discussion/api.js"
```
Comment (gustavpursche) [inline tools/es5to6.json]: Please assign these 2 to me.

File: tools/es5to6.json [hunk 4]
```diff
@@ -1,121 +1,112 @@
...
+  "Simon Adcock": [
+    "bootstraps/enhanced/recipe-article.js",
+    "lib/picturefill.js",
     "projects/common/modules/crosswords/constants.js",
     "projects/common/modules/crosswords/controls.js",
     "projects/common/modules/crosswords/crossword.js",
     "projects/common/modules/crosswords/grid.js",
-    "projects/common/modules/crosswords/helpers.js"
+    "projects/common/modules/crosswords/helpers.js",
+    "projects/common/modules/discussion/discussion-frontend.js",
```
Comment (gustavpursche) [inline tools/es5to6.json]: This one can also be mine.

File: tools/es5to6.json (hunk 3 above)
Comment (SiAdcock) [inline tools/es5to6.json]: Cool, I'll bump all your discussion stuff to the top since you're on a roll! (but feel free to amend the order as you see fit)

**Review by gu-stav — APPROVED** [review]
💥

**Comment by philwills** [comment]
Don't suppose @sndrs or @SiAdcock fancies swapping `projects/common/modules/gallery/lightbox.js` for a couple of crossword files? 

I have an attachment to crosswords and haven't had any excuse to touch the code for them for a few years.

**Comment by SiAdcock** [comment]
Couple of juicy crossword modules coming your way @philwills 😄

**Pull request merged**